data/targets.yml

```diff
@@ -31,6 +31,11 @@
   obs_api: "https://api.opensuse.org"
   obs_project: "YaST:SLE-12:SP2"
   obs_target: "SLE_12_SP2"
+:sle12sp2_casp:
```

Comment:
What about simple `casp10` ? It's shorter to type. :wink: The fact that it's based on SP2 is really not important here I think...